app/scripts/templates/index.mustache

```diff
@@ -0,0 +1,34 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-enter-email-header">
+      {{#serviceName}}
+        <!-- L10N: For languages structured like English, the second phrase can read "to continue to %(serviceName)s" -->
+        {{#t}}Enter your email{{/t}} <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+      {{/serviceName}}
+      {{^serviceName}}
+        {{#t}}Enter your email{{/t}}
+      {{/serviceName}}
+    </h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    {{{ syncSuggestionHTML }}}
```

Comment:
Good point. Until yesterday, this behavior wasn't limited to Sync, anybody that went to the `/email` page. I received some early feedback from @rfk that using a new endpoint didn't seem as awesome as it should be, so I migrated all the code into index.js and isolated the behavior to Sync. I suppose Sync needn't be a restriction, but it seems like an easy way to help us A/B test ensuring a homogeneous group.

I'll remove the Sync suggestion stuff until we need it.